### DISCUSSION TITLE
Sharing Tools, Assistants, Collections, Models, and Files

### DISCUSSION THREAD

**Discussion by WatchfulEyeOfZod** [mckaywrigley/chatbot-ui#1858] [Help]
Is there a wait to create tools (or files or collections) that are shared across multiple users? For example, define a tool and update the database entry to make it "Shared" so that it exists for all users automatically when then log into the chatbotUI?

For example, in the `tools` table in supabase, there is a column for sharing with a default vaue of "private". Could this be adjusted to a value of "shared" or "public" so that it shows up for all users?

**Comment by DunaSpice** [comment]
would be nice. Lets collab
